package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "taskcluster-babel",
+  "version": "0.0.1",
+  "description": "babel executable for taskcluster projects",
+  "main": "lib/tc-babel.js",
+  "scripts": {
+    "test": "mocha test/*_test.js"
+  },
+  "bin": {
+    "tc-babel": "./bin/cli.js"
```

Comment:
so, calling the package `sane-babel-cli` and this `sane-babel` would be fine :)

Also you forgot to `git add bin/cli.js`